Dockerfile.dapper

```
@@ -85,8 +91,12 @@ RUN wget -O - ${YQ_URL} > /usr/bin/yq && chmod +x /usr/bin/yq
 ENV KUBECTL_URL=https://storage.googleapis.com/kubernetes-release/release/v1.21.0/bin/linux/${ARCH}/kubectl
 RUN wget -O - ${KUBECTL_URL} > /usr/bin/kubectl && chmod +x /usr/bin/kubectl
 
-RUN apt-get update && \
-    apt-get install -y tox python3.8 python3-dev python3.8-dev libffi-dev libssl-dev
+RUN if [ -x /usr/bin/zypper ]; then \
+     zypper install -y tox python39 python39-devel libffi-devel libopenssl-devel ; \
```

```suggestion
     zypper install -y python3-tox libffi-devel libopenssl-devel ; \
```
no need for python3.9 and use proper name for tox package

---

thanks, manually merged